invoices_app/tests.py

```diff
@@ -1,6 +1,7 @@
 from parameterized import parameterized
 from django.test import TestCase
-from .models import Company, BankAccount, TaxPayer, TaxPayerState, TaxPayerArgentina, Address
+from .models import Address, BankAccount, Company, CompanyUserPermission, TaxPayer, TaxPayerArgentina, TaxPayerState
```

Comment:
I think that a multiline import could make it more readable! What do you think about it?
Also, specifying the app could help too
`from invoices_app.models import (...`